dash/src/memory/HostSpace.cc

```diff
@@ -0,0 +1,15 @@
+#include <dash/memory/HostSpace.h>
```

Comment:
Out of curiosity: why are the memory spaces not implemented in the header? Is there some significant compilation overhead?

Reply:
Because it does not rely on templates. ;)